src/components/app/details/appDetails/GraphsModal.tsx

```diff
@@ -218,6 +220,26 @@ export class GraphModal extends Component<GraphModalProps, GraphModalState>{
         });
     }
 
+    handleLatencyChange(selected): void {
+        let k8sVersion = this.props.k8sVersion;
```

Comment:
[Refactoring] refactoring required as most of things same in handleStatusChange function.  
However can be done later.